### DISCUSSION TITLE
Posição da Notificação gerada pelo Http Interceptor

### DISCUSSION THREAD

**Discussion by ademario** [po-ui/po-angular#1785] [Tire suas dúvidas] [answered]
Quando o Http Interceptor identifica uma mensagem de resposta no retorno da requisição, é gerada uma notificação que por padrão é exibida na parte de baixo da tela. É possivel mudar  essa notificação para o topo da tela, por padrão?

**Comment by felipepetuco** [comment]
Boa tarde, há uma interface para https://po-ui.io/documentation/po-notification com propriedade "orientation": posição da notificação na página que pode ser Top (topo) ou Bottom(rodapé). A posição padrão é bottom.

**Comment by ademario** [comment]
Boa Tarde. Esta interface eu já faço uso quando eu gero as notificações. Exemplo:

constructor(public poNotification: PoNotificationService) {
  }
showNotification(){
   this.poNotification.success({ message: 'Sucesso', orientation: PoToasterOrientation.Top });
}

No caso em questão, as mensagens vem do endpoint requisitado conforme o padrao indicado no Http Intercept
https://po-ui.io/documentation/po-http-interceptor  (Ver Estrutura das mensagens)

Nesse caso, a notificação é gerada pelo framework, sem precisar instanciar o poNotification. Neste caso sempre está vindo na parte de baixo.. teria como alterar esse comportamento?

**Comment by anliben** [comment, marked as answer]
Boa Tarde @ademario, tudo bem?

Atualmente não temos uma solução para modificar a posição do toast do notification quando ele vem do http-interceptor!

Attr.
